packages/@react-spectrum/numberfield/test/NumberField.test.js

```diff
@@ -961,10 +985,12 @@ describe('NumberField', function () {
     ${'French Euros'} | ${{formatOptions: {style: 'currency', currency: 'EUR'}}} | ${'fr-FR'} | ${['10,00 €', '11,00 €', '9,00 €', '9,00 €']}
     ${'Arabic Euros'} | ${{formatOptions: {style: 'currency', currency: 'EUR'}}} | ${'ar-AE'} | ${['١٠٫٠٠ €', '١١٫٠٠ €', '٩٫٠٠ €', '٩٫٠٠ €']}
   `('$Name pressing up arrow & down arrow keeps focus state formatting', ({props, locale, expected}) => {
-    let {textField} = renderNumberField({defaultValue: 10, ...props}, {locale});
+    let {textField} = renderNumberField({defaultValue: 10, onKeyDown: onKeyDownSpy, onKeyUp: onKeyUpSpy, ...props}, {locale});
 
     act(() => {textField.focus();});
     expect(textField).toHaveAttribute('value', expected[0]);
+    expect(onKeyDownSpy).not.toHaveBeenCalled();
+    expect(onKeyDownSpy).not.toHaveBeenCalled();
```

Comment:
what were these added for?

Reply:
It should have been up and down spys, confirming that they aren't called before interactions happen.